chainstate/src/detail/transaction_verifier/mod.rs

```diff
@@ -201,60 +215,98 @@ impl<'a, S: BlockchainStorageRead> TransactionVerifier<'a, S> {
                     .ok_or(ConnectTransactionError::MissingOutputOrSpent)?;
                 entry.insert(CachedInputsOperation::Read(tx_index));
             }
-        }
+        };
```

Comment:
Why?

Reply:
Fixed. I had faced semicolon addings a few times, probably some glitch in cargo-fmt.